packages/time/src/core/weekInfoPolyfill.ts

```diff
@@ -0,0 +1,1864 @@
+interface WeekInfo {
+  firstDay: number
+  weekend: number[]
+  minimalDays: number
+}
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace Intl {
+  interface Locale {
+    getWeekInfo: () => WeekInfo
+    weekInfo?: WeekInfo
+  }
+}
+
+;(function () {
+  if (typeof (Intl as any).Locale.prototype.getWeekInfo !== 'function') {
```

Comment:
Safari is the only browser (currently) with `Locale.prototype.getWeekInfo`. Chrome and Edge have a `weekInfo` accessor of `Locale`. Those browsers with the accessor should return it's value when calling `Local.prototype.getWeekInfo`, whereas those that do not (Firefox) would need the true polyfil method.

Since the polyfil is only applied to browsers that don't have `getWeekInfo`, and a true override method is only applied to one's without `Locale.weekInfo`, it would be good to dynamically load the additional data only to those browsers that require it.